docs/interpreter/sap.md

```diff
@@ -121,3 +121,21 @@ where
 
   and [Folder1].[Dimension5] in ('Value1', 'Value2');
 ```
+
+## Object Interpolation
+The SAP interpreter also supports interpolation of `ZeppelinContext` objects into the paragraph text.
+To enable this feature set `universe.interpolation` to `true`. The following example shows one use of this facility:
+
+####In Scala cell:
+
+```scala
+z.put("curr_date", "2018-01-01 00:00:00")
```

Comment:
In you scenario, do you just need scala repl or needs spark support ? The reason I ask is that I am thinking whether we should introduce pure scala interpreter

Reply:
Tinkoff team use only spark (pyspark) interpreter.
I don't feel the necessary of pure scala interpreter.